ci/rpm/install_maldet.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -uex
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+# at some point we want to use: shellcheck source=ci/rpm/distro_info.sh
+# shellcheck disable=SC1091
+source "$mydir/distro_info.sh"
+
+: "{BUILD_URL:=}"
+if command -v dnf; then
+  dnf --assumeyes install clamav clamav-devel
+  if [ -n "${BUILD_URL}" ]; then
+    repo_url="${BUILD_URL}artifact/artifacts/${PUBLIC_DISTRO}${MAJOR_VERSION}/"
+    dnf --assumeyes config-manager --add-repo="$repo_url"
+    repo="${repo_url#*://}"
+    repo="${repo//%/}"
+    repo="${repo//\//_}"
+    # bug in EL7 DNF: this needs to be enabled before it can be disabled
```

Comment:
We do not care about EL7.

Reply:
As above, this will be cherrypicked into branches that are still supporting el7.

We are still building rpms in master for el7.